doc/man7/EVP_KDF-KRB5KDF.pod

```diff
@@ -0,0 +1,119 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF-KRB5KDF - The RFC3961 Krb5 KDF EVP_KDF implementation
+
+=head1 DESCRIPTION
+
+Support for computing the B<KRB5KDF> KDF through the B<EVP_KDF> API.
+
+The EVP_KDF-KRB5KDF algorithm implements the key derivation function defined
+in RFC 3961, section 5.1 and is used by Krb5 to derive session keys.
+Three inputs are required to perform key derivation: a cipher, (for example
+AES-128-CBC), the initial key, and a constant.
+
+=head2 Identity
+
+"KRB5KDF" is the name for this implementation;
+it can be used with the EVP_KDF_fetch() function.
+
+=head2 Supported parameters
+
+The supported parameters are:
+
+=over 4
+
+=item B<OSSL_KDF_PARAM_PROPERTIES> ("properties") <UTF8 string>
+
+=item B<OSSL_KDF_PARAM_CIPHER> ("cipher") <UTF8 string>
+
+=item B<OSSL_KDF_PARAM_KEY> ("key") <octet string>
+
+These parameters work as described in L<EVP_KDF(3)/PARAMETERS>.
+
+=item B<OSSL_KDF_PARAM_CONSTANT> ("constant") <octet string>
+
+This parameter sets the constant value for the KDF.
+If a value is already set, the contents are replaced.
```

Comment:
That was... rather terse.  For someone not in the know (such as myself), this doesn't really explain anything.

Reply:
Sorry , how could I improve it ?
Ths description doesn't look to me vaguer than the description of the SALT parameter.

Reply:
It's true...  I guess that when it comes to salts, I'm in the know, so I don't react and it's possible that I should.  But I just read through RFC 3961 and became a little wiser.  So er, I'm letting this go for now.